promptsource/templates/quac/templates.yaml

```diff
@@ -0,0 +1,90 @@
+dataset: quac
+templates:
+  12c9d007-991c-49ed-82e2-13a7d3147881: !Template
+    answer_choices: null
+    answer_choices_key: null
+    id: 12c9d007-991c-49ed-82e2-13a7d3147881
+    jinja: "Given the dialogue : \n\nStudent: {{questions[0]}}\n\nTeacher: {{answers.texts[0]\
+      \ | choice}}\n\nThe context : {{context}}\n\nAnswer the question: {{questions[1]\
+      \ }}\n|||\n{{answers.texts[1] | choice }}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - Squad
+      original_task: true
```

Comment:
I don't think this is the task?

Reply:
I think this template doesn't make sense. Will add better ones in the next commit.